source/docs/03-metadata.xml

```diff
@@ -1392,9 +1392,19 @@
                         <specDesc key="colophon"/>
                      </specList>
                      </p>
-                  <p>The <gi scheme="MEI">condition</gi> element may contain text, or child elements that may be used for structuring the description (e.g., for linking) to accurately describe the physical state or condition of a source.</p>
-                     <p>The condition can thus be described in general (<gi scheme="MEI">physDesc</gi>) or in specific contexts (see “contained by” section of the element definition).</p>
-                     <p>For describing the condition of parts of a source (e.g., the binding) the <gi scheme="MEI">condition</gi> element can be used on the specific level. The level of detail you provide will depend on your specific needs for accurately recording the material condition.</p>
+                  <p>Encoding general information about the condition of a source.</p>
+                   <p>
```

Comment:
Please check indentation and line breaks of all the blocks with

```
<p><specList>
 ...
</specList></p> 
```

Should be:


```
<p>
    <specList>
 ...
    </specList>
</p> 
```

Reply:
I think we should do a general format and indent when preparing the release, so this might be ok for now